docs/guides/08-introduction-to-image-files.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction to Image Markers
 
-CARTO VL supports the use of images to render point data. Image markers can be accessed directly from CARTO VL's built-in symbol library, or from external sources via an HTTP request. Any image (built-in or external) can be colored, sized and placed using CARTO VL expressions.
+CARTO VL supports the use of images to render point data. Image files can be accessed directly from CARTO VL's built-in symbol library, or from external sources via an HTTP request. Any image (built-in or external) can be colored, sized and placed using CARTO VL expressions.
```

Comment:
Will we have a reference/URL for the built-in image files? I think we should and also put that link here.